app/controllers/v3/service_route_bindings_controller.rb

```diff
@@ -50,11 +57,21 @@ def index
       paginated_result: SequelPaginator.new.get_page(route_bindings, message.try(:pagination_options)),
       path: '/v3/service_route_bindings',
       message: message,
+      decorators: decorators(message)
     )
   end
 
   private
 
+  AVAILABLE_DECORATORS = [
+    IncludeBindingServiceInstanceDecorator,
+    IncludeRouteDecorator
```

Comment:
Would probably rename this to `IncludeBindingRouteDecorator`to be in line with other decorators